config.go

```diff
@@ -95,6 +104,8 @@ func (c *Config) Validate() error {
 	switch {
 	case c.Group.Offsets.Retry.Max < 0:
 		return sarama.ConfigurationError("Group.Offsets.Retry.Max must be >= 0")
+	case c.Group.Offsets.Synchronization.DwellTime == nil || *(c.Group.Offsets.Synchronization.DwellTime) > time.Minute*10:
```

Comment:
Not 100% sure, why 10 minutes? Shouldn't it be less than the `c.Group.Session.Timeout`?